### PR TITLE
Added the daily load formula

### PR DESCRIPTION
**Pull request by Expertium**
https://www.reddit.com/r/Anki/comments/1h6n9dk/comment/m0fza4c/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button

Originally I wanted to add the formula, but sorata told me not to. Seeing that at least once more person wants to see it, I think we should add it.

### CODE REVIEW

**Comment by brishtibheja** [comment]
It's a sample of a few selected person wanting this, but if it's to be done, I think we can make it cleaner using mathjax.

In `book.toml`, have `mathjax-support = true` to enable mathjax.

https://github.com/ankitects/anki-manual/blob/bb51e749f4f33ac8395693ae964e5ee9d088b97a/book.toml#L16-L20

Then format the equation using mathjax syntax. Using `\\[`, `\\]` as delimiters, you'd write:

```
\[\frac{1}{I_1} + \frac{1}{I_2} + \frac{1}{I_3} + \dots + \frac{1}{I_n}\]
```

This renders as: 

$$\frac{1}{I_1} + \frac{1}{I_2} + \frac{1}{I_3} + \dots + \frac{1}{I_n}$$
___

Ref: https://rust-lang.github.io/mdBook/format/mathjax.html

**Comment by Expertium** [comment]
I don't know how to do that. Feel free to PR.

File: src/stats.md [hunk 1]
```diff
@@ -117,6 +117,14 @@ Daily load is an estimate of the average number of cards to be reviewed daily.
 If you don't have a backlog, daily load should be approximately equal to 
 your number of due cards.
 
+It is calculated as follows:
+$$\frac{1}{I_1} + \frac{1}{I_2} + \frac{1}{I_3} + \dots + \frac{1}{I_n}$$
```
Comment (brishtibheja) [inline src/stats.md:121]:
```suggestion
\\[\frac{1}{I_1} + \frac{1}{I_2} + \frac{1}{I_3} + \dots + \frac{1}{I_n}\\]
```

File: src/stats.md [hunk 2]
```diff
@@ -117,6 +117,14 @@ Daily load is an estimate of the average number of cards to be reviewed daily.
...
+where In is the interval of
```
Comment (brishtibheja) [inline src/stats.md:122]:
```suggestion

Here, \\(I_n\\) is the interval of
```

File: src/stats.md [hunk 3]
```diff
@@ -117,6 +117,14 @@ Daily load is an estimate of the average number of cards to be reviewed daily.
...
+the n-th card. If the interval is less than one day, the summation term is 1. This
+prevents cards with short intervals from unrealistically skewing the value of daily load.
+Example: you have a card with an interval of 10 days and a card with an interval of 50 days.
```
Comment (brishtibheja) [inline src/stats.md:125]: how about a newline here.
```suggestion

Example: you have a card with an interval of 10 days and a card with an interval of 50 days. 
```

File: src/stats.md [hunk 4]
```diff
@@ -117,6 +117,14 @@ Daily load is an estimate of the average number of cards to be reviewed daily.
 If you don't have a backlog, daily load should be approximately equal to 
 your number of due cards.
 
+It is calculated as follows:
+$$\frac{1}{I_1} + \frac{1}{I_2} + \frac{1}{I_3} + \dots + \frac{1}{I_n}$$
+where In is the interval of
+the n-th card. If the interval is less than one day, the summation term is 1. This
+prevents cards with short intervals from unrealistically skewing the value of daily load.
+Example: you have a card with an interval of 10 days and a card with an interval of 50 days. 
+Daily load = \\( ]frac{1}{10} + \frac{1}{50} = 0.12 \\), meaning that, *on average*, you will have 0.12 due cards per day.
```
Comment (brishtibheja) [inline src/stats.md:126]:
```suggestion
Daily load = \\(\frac{1}{10} + \frac{1}{50} = 0.12\\), meaning that, _on average_, you will have 0.12 due cards per day.
```

**Review by brishtibheja — CHANGES_REQUESTED** [review]

**Comment by Expertium** [comment]
@dae

**Comment by dae** [comment]
Thank you both, and sorry about the delay.

**Pull request merged**